compliance/components/ManualProcedures/component.yaml

```diff
@@ -9,7 +9,7 @@ satisfies:
   narrative:
     - text: |
         XXX You will need to document your account management policies and procedures
-        for GitHub and GCP and your app's users.  There is an excellent 
+        for GitHub, cloud.gov, and your app's users.  There is an excellent 
         [GSA Account Management Procedures](https://docs.google.com/document/d/1TiqDLPnANT5aPuDUXbj2fR3iYCBqpO1WE6urP8IF0kA/edit)
```

Comment:
Given that this will be a HHS ATO, do we want to remove the references to the GSA LATO?

Reply:
Until we get the HHS ATO package actually given to us formally with directions on how to fill it out, this process and what needs to be documented is all kinda speculation.  I figured I'd just leave the LATO work I did in there as a placeholder until we got that, because the process is probably going to be similar-ish.